### ISSUE TITLE
Failing to build pillow

### ISSUE THREAD

**Issue by Chrisrdouglas** [Baekalfen/PyBoy#45]
I tried following the instructions for Ubuntu. I ran `pip install -r requirements.txt` and it failed to build pillow

**Comment by Baekalfen** [comment]
Can you post the error message?

**Comment by Chrisrdouglas** [comment]
I tried to repeat what i did yesterday and it worked this time...

**Comment by Baekalfen** [comment]
Glad to hear it works :) Did you maybe install `build-essential` through apt? Maybe it should be in the instructions.

**Comment by Chrisrdouglas** [comment]
Nah, I didn't install `build-essential` through apt. I go back to work on Monday and i'll try to see if i can repeat the error on one of the linux computers there.

Also, i tried using the getScreenBuffer() method to get the numpy array with the screen buffer and it gave me a `PyBoy.GameWindow.GameWindow_SDL2.SdlGameWindow object`. Want me to open another issue for that?

**Comment by Baekalfen** [comment]
Thank you. I really want the installation instructions to be thorough.

Yes, please. At any issue or inconsistency, just open an issue. If it's nothing, we can just close it again :)
